app/src/util/helpers.ts

```diff
@@ -156,3 +156,38 @@ export function findClosestYear(
     null as PopulationEntry | null,
   );
 }
+
+export function findClosestYearToInventory(
```

Comment:
This should already by in develop, right?
Or are we backporting changes from main here?